frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.tsx

```diff
@@ -81,27 +93,22 @@ export const CommandPalette: FC = () => {
           <Command.Empty>No results found.</Command.Empty>
           <Command.Group heading="Tables">
             {Object.values(schema.current.tables).map((table) => (
-              <Command.Item
-                key={table.name}
-                value={table.name}
-                onSelect={() => goToERD(table.name)}
-                data-focused={focusedTableName === table.name}
-              >
-                {/** biome-ignore lint/a11y/useKeyWithClickEvents: Keyboard interaction is implemented in the parent Command.Item component's onSelect handler. */}
-                <div
-                  className={styles.itemInner}
+              <Command.Item key={table.name} value={table.name} asChild>
+                <a
+                  href={`?active=${table.name}`}
```

Comment:
This href value will omit other search parameters when a new tab opens.
Let me fix it in another PR to make this PR smaller.